src/RoleObjects/OptionData.js

```diff
@@ -1,18 +1,19 @@
-import AccessibilityObject from './AccessibilityObject.js';
+import AccessibilityObject from './AccessibilityObject';
 
 export default class OptionData extends AccessibilityObject {
-  addChild(displayObject) {
+  addChild(displayObject) { // eslint-disable-line
     throw new Error(`${this.role} cannot have children`);
   }
 
-  addChildAt(displayObject, index) {
+  addChildAt(displayObject, index) { // eslint-disable-line
```

Comment:
Please add which rule(s) need to be disabled instead of disabling all linting for this line.